CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
```

Comment:
We don't use CHANGELOG in this library. We can track changes in the release part of Github